core/tests/behavior/async_rename.rs

```diff
@@ -178,6 +178,11 @@ pub async fn test_rename_nested(op: Operator) -> Result<()> {
 
 /// Rename to a exist path should overwrite successfully.
 pub async fn test_rename_overwrite(op: Operator) -> Result<()> {
+    // Dropbox does not support rename overwrite.
```

Comment:
We can remove existing file before rename/copy.

Reply:
Updated.